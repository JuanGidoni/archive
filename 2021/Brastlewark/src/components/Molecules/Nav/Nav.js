import NavList from '../../Atoms/NavList';

const Nav = () => {
 return (
  <ul className="navbar-nav ml-auto">
   <NavList text="Age" to="/filter/age/a" />
   <NavList text="Weight" to="/filter/weight/a" />
   <NavList text="Height" to="/filter/height/a" />
  </ul>
 );
};

export default Nav;
