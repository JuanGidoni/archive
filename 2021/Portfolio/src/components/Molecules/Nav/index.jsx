import { Link } from 'react-router-dom';
import { FaFileDownload } from 'react-icons/fa';
import './index.css';
import CV from '../../Assets/CV_Gidoni_Juan_2021.pdf';

const Nav = ({ toggleMenu, showMenu }) => {
 return (
  <>
   <nav id="navbar">
    <div className="nav-wrapper">
     <Link to="/">
      <p className="brand">
       juan
       <strong>gidoni</strong>
      </p>
     </Link>
     <div className="flex flex-center flex-row center">
      <button
       onClick={toggleMenu}
       className={showMenu === 'active' ? 'menu-button active' : 'menu-button'}
      >
       <span />
      </button>
      <div
       className={showMenu === 'active' ? 'd-none' : 'pl-3'}>
       <a href={CV}
        className="center flex flex-center flex-row">
        CV<FaFileDownload />
       </a>
      </div>
     </div>
    </div>
   </nav>
  </>
 );
};
export default Nav;