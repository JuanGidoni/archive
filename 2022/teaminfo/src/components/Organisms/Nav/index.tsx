import { useContext } from "react";
import DataContext from "../../Context/DataContext";
import { Collapse, UnOrderList } from "../../Molecules";
import { BasicPropsComponent } from "../../Types";

const Nav = ({ className }: BasicPropsComponent) => {
  const { navList } = useContext<any>(DataContext);
  return (
    <nav className={className}>
      <Collapse />
      <div className="collapse navbar-collapse" id="navbarSupportedContent">
        <UnOrderList className="navbar-nav mr-auto" items={navList} />
      </div>
    </nav>
  );
};
export default Nav;
