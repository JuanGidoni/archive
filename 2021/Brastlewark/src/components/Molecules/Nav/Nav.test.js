import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Nav from './Nav';

describe('<Nav />', () => {
 it('renders correctly and match snapshot', () => {
  const tree = renderer
   .create(<Router><Nav /></Router>)
   .toJSON();
  expect(tree).toMatchSnapshot();
 });
});