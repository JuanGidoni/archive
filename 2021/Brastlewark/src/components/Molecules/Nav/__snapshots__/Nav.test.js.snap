// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav /> renders correctly and match snapshot 1`] = `
<ul
  className="navbar-nav ml-auto"
>
  <li
    className="nav-item"
  >
    <a
      className="nav-link "
      href="/filter/age/a"
      onClick={[Function]}
    >
      Age
    </a>
  </li>
  <li
    className="nav-item"
  >
    <a
      className="nav-link "
      href="/filter/weight/a"
      onClick={[Function]}
    >
      Weight
    </a>
  </li>
  <li
    className="nav-item"
  >
    <a
      className="nav-link "
      href="/filter/height/a"
      onClick={[Function]}
    >
      Height
    </a>
  </li>
</ul>
`;
